gh-69605: Add math.integer to PyREPL module completer hardcoded list (#144811)
python/cpython: Misc/NEWS.d/next/Core_and_Builtins/2026-02-14-13-07-08.gh-issue-69605.4aL4hn.rst

> Fix :mod:`math.integer` not being suggested in the :term:`REPL` auto-completion of imports.

Reviewer: The future person reading might like this a bit more:

```suggestion
Add :mod:`math.integer` to :term:`REPL` auto-completion of imports.
```
